### ISSUE TITLE
(CV)  compare computed checksums against online moab checksums (blocked by #39)

### ISSUE THREAD

**Issue by ndushay** [sul-dlss/preservation_catalog#40]
blocked by #39

Note: checksums = MD5, SHA1, SHA256

Checksum validation will do the following:

given an online Moab:

1.  compute checksums for every file;  probably store as a hash keyed by version;  keyed by filename (for each dir) -- See #518

2.  for each VERSION:
  - compare computed checksums of manifestInventory.xml against (TCR) stored checksums  -- see #520
   - compare computed checksums of other files in "manifest" directory against what is in manifestInventory.xml file -- see #521
  -  compare computed checksums of files in "data" directory against what is in signatureCatalog.xml -- see #522

NOTE:  signatureCatalog.xml has checksum information for its version and all previous versions (can decrease IO?)

**Comment by ndushay** [comment]
Note there is a lot of "verification" and "validation" code in moab-versioning that might be useful here.  storage_object, storage_object_version, content_inventory, file_inventory, etc.

**Comment by ndushay** [comment]
Closing as duplicate of #518, #520, #521, #522
